components/friend.tsx

```diff
@@ -8,40 +8,33 @@ const friends = [
     source: 'https://github.com/JarbasHiveMind',
     logo: 'https://github.com/JarbasHiveMind/HiveMind-assets/raw/master/logo/hivemind-512.png'
   },
-  {
-    name: 'Plasma Bigscreen',
-    description: 'Plasma Bigscreen integrates and uses OpenVoiceOS as voice framework stack to serve voice queries and voice applications (skills with a homescreen), one can easily enable mycroft / ovos integration in the bigscreen launcher by installing ovos core and required services and enabling the integration switch in the bigscreen KCM.',
-    website: 'https://plasma-bigscreen.org',
-    source: 'https://invent.kde.org/plasma/plasma-bigscreen',
-    logo: 'https://plasma-bigscreen.org/img/logo.png',
-    screenshot: 'https://plasma-bigscreen.org/img/screenshot-1.png'
-  },
   {
     name: 'NeonGecko',
     description: 'Neon was one of the first projects ever to adopt ovos-core as a library to build their own voice assistant, Neon works closely together with OVOS and both projects are mostly compatible.',
     website: 'https://neon.ai',
     source: 'https://github.com/NeonGeckoCom',
     logo: 'https://neon.ai/wp-content/uploads/2024/04/neon-ai-artificial-intelligence-2-2-3.png'
   },
+  {
+    name: 'Koninklijke Visio',
+    description: 'Visio offers information and advice, but also various forms of research, guidance, rehabilitation, education, work and living. These services are available for people who are visually impaired or blind, even if they also have an intellectual, physical or other sensory disability.',
+    website: 'http://www.visio.org',
+    source: 'https://github.com/timonvanhasselt',
+    logo: 'https://www.visio.org/media/klhfdhpp/visio-logo.svg'
+  },
   {
     name: 'Mycroft',
     description: 'Mycroft AI started it all, it was one of the first ever FOSS voice assistants and is the project OVOS descends from. Most applications made for mycroft will work in OVOS and vice-versa.',
     website: 'https://mycroft.ai',
     source: 'https://github.com/MycroftAI',
-    logo: 'https://mycroft.ai/wp-content/uploads/2017/02/Mycroft-Twitter-Profile.jpg'
+    logo: 'https://yt3.googleusercontent.com/ytc/AIdro_l_P3GVwHPSSihmb__A5hlkNnO4ZGjdG3SmUk17f-Az2Is=s900-c-k-c0x00ffffff-no-rj'
```

Comment:
we probably should host a dedicated svg file, this was a quick hack to make it work with a pic from a google image search

Reply:
@NeonDaniel do you have a proper svg file available?

Reply:
nvm, used their github logo